service/dap/server_test.go

```diff
@@ -4239,12 +4468,65 @@ Type 'dlv help' followed by a command for full documentation.
 						t.Errorf("\ngot: %#v, want sources=\"\"", got)
 					}
 
+					if runtime.GOOS == "freebsd" || runtime.GOOS == "darwin" {
```

Comment:
Invert the tests and put the follow exec tests inside the if, it's too easy to miss this and add stuff to this test that will be skipped on freebsd and darwin (there's already some of that).